source/agora/consensus/protocol/Nominator.d

```diff
@@ -1031,24 +1031,86 @@ extern(D):
     public override Value combineCandidates (uint64_t slot_idx,
```

Comment:
The doc of this function should explain the way how a candidate will be selected based on new conditions like `missing_validators`, `fee`, and `hash`. Could you add it to the doc?

Reply:
> The doc of this function should explain the way how a candidate will be selected based on new conditions like `missing_validators`, `fee`, and `hash`. Could you add it to the doc?

added